tests/console/commands/self/test_add_plugins.py

```diff
@@ -138,6 +138,53 @@ def test_add_with_git_constraint_with_extras(
     )
 
 
+@pytest.mark.parametrize(
+    "url, rev",
+    [
+        ("git+https://github.com/demo/poetry-plugin2.git#subdirectory=subdir", None),
+        (
+            "git+https://github.com/demo/poetry-plugin2.git@master#subdirectory=subdir",
+            "master",
```

Comment:
```suggestion
            "master"
```

Let's drop the trailing comma so this can be formatted better.

Reply:
This tuple is too long for Black to format on a single line, even without the trailing comma.

Reply:
Fair enough; I'd just drop it and leave the formatting alone as a matter of personal preference.

Reply:
If you drop the trailing comma, Black just re-adds it.